MCprep_addon/materials/prep.py

```diff
@@ -247,13 +247,13 @@ def execute(self, context):
 
 			if engine == 'CYCLES' or engine == 'BLENDER_EEVEE':
 				options = generate.PrepOptions(
-					passes=passes,
-					use_reflections=self.useReflections,
-					use_principled=self.usePrincipledShader,
-					only_solid=self.makeSolid,
-					pack_format=self.packFormat,
-					use_emission_nodes=self.useEmission,
-					use_emission=False  # This is for an option set in matprep_cycles
+					passes, 
+					self.useReflections, 
+					self.usePrincipledShader, 
+					self.makeSolid, 
+					generate.PackFormat.from_str(self.packFormat.upper()), 
```

Comment:
`from_str` is unnecessary, just do `generate.PackFormat[self.packformat.upper()]`. It's already a documented feature of Python.